src/safeds/ml/classical/classification/_baseline_classifier.py

```diff
@@ -0,0 +1,185 @@
+import copy
+from concurrent.futures import ALL_COMPLETED, wait
+from typing import Self
+
+from safeds._validation._check_columns_are_numeric import _check_columns_are_numeric
+from safeds.data.labeled.containers import TabularDataset
+from safeds.exceptions import (
+    DatasetMissesDataError,
+    DatasetMissesTargetError,
+    FeatureDataMismatchError,
+    ModelNotFittedError,
+)
+from safeds.ml.classical.classification import (
+    AdaBoostClassifier,
+    Classifier,
+    DecisionTreeClassifier,
+    GradientBoostingClassifier,
+    RandomForestClassifier,
+    SupportVectorClassifier,
+)
+
+
+def _fit_single_model(model: Classifier, train_data: TabularDataset) -> Classifier:
+    return model.fit(train_data)  # pragma: no cover
+
+
+def _predict_single_model(model: Classifier, test_data: TabularDataset) -> TabularDataset:
+    return model.predict(test_data)  # pragma: no cover
+
+
+class BaselineClassifier:
```

Comment:
is there still a need to differ between a classifier and a regression?

Reply:
The only differences between them are the list of Models and the list of metrics. We do need the user to explicitly tell us which one to use, as we currently do not have a way to differentiate the two only by the given data. We could change the current implementation to just one Baseline Class and add a parameter to define the use case but given our other model implementations I would say it makes sense to keep two classes for consistency.